services/processor/native/test/harness.go

```diff
@@ -110,9 +112,9 @@ func (h *harness) expectSdkCallMadeWithAddressGetCaller(returnAddress []byte) {
 			input.OperationName == native.SDK_OPERATION_NAME_ADDRESS &&
 			input.MethodName == "getCallerAddress"
 	}
-
+	outputArgs, _ := protocol.ArgumentsFromNatives(builders.VarsToSlice(returnAddress)) // err ignored on purpose
```

Comment:
why on purpose?

Reply:
because in many of the tests we "hard code" create this with a known type (string or []byte) it is unlikely that we will stop supporting this type so the error is unlikely to happen. 
i didn't want to panic in a case that is really unlikely.

Reply:
so explain this in the comment